libraries/core_libs/storage/src/storage.cpp

```diff
@@ -217,6 +217,23 @@ void DbStorage::rebuildColumns(const rocksdb::Options& options) {
     const auto it = std::find_if(Columns::all.begin(), Columns::all.end(),
                                  [&handles, i](const Column& col) { return col.name() == handles[i]->GetName(); });
     if (it == Columns::all.end()) {
+      // All the data from dag_blocks_index is moved to dag_blocks_level
+      if (handles[i]->GetName() == "dag_blocks_index") {
+        rocksdb::ColumnFamilyHandle* handle_dag_blocks_level;
+
+        auto options = rocksdb::ColumnFamilyOptions();
+        options.comparator = getIntComparator<uint64_t>();
+        checkStatus(db->CreateColumnFamily(options, Columns::dag_blocks_level.name(), &handle_dag_blocks_level));
+
+        auto it_dag_level = std::unique_ptr<rocksdb::Iterator>(db->NewIterator(read_options_, handles[i]));
+        it_dag_level->SeekToFirst();
+
+        while (it_dag_level->Valid()) {
+          checkStatus(db->Put(write_options_, handle_dag_blocks_level, toSlice(it_dag_level->key()),
+                              toSlice(it_dag_level->value())));
+          it_dag_level->Next();
+        }
+      }
```

Comment:
I initially tried doing this witin a migration but I could not make it work due to rocksdb API limitations. The problem is that I am not changing the format of the data in the existing column like for other migrations but I am changing the column itself. Migration as currently implemented is happening after all of the columns are already opened. I needed to do this before the database being opened similar to how delete unused columns. This is really fast. For mainnet it is under 30 seconds. I could not find a way to change the existing column definition similar to what we do for migration since rocksdb API insists that all the columns are present when opening the db.